tf2onnx/shape_inference.py

```diff
@@ -9,18 +9,219 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 import logging
-import numpy as np
-from onnx import onnx_pb
+import os
+from distutils.version import LooseVersion
+

```

Comment:
revisit make_sure usage, some should throw, some may just swallow (maybe log a warning).